routes/routes.js

```diff
@@ -6,6 +6,8 @@ const { check, validationResult} = require("express-validator");
 const bcrypt = require("bcryptjs");
 const config = require('../config');
 
+const authController = require('../controllers/authController');
```

Comment:
You could imagine this file would become very condensed down, with just loading the controllers in and defining the routes passing the handlers in.